docs/README.md

```diff
@@ -0,0 +1,112 @@
+    }
+    # module.cis_dns_records.local_file.dns_record_file[0] must be replaced
+    -/+ resource "local_file" "dns_record_file" {
+      ~ filename             = "../../modules/dns/dns_records_2023-12-10T09:29:24Z.txt"-> (known after apply) # forces replacement
+      ~ id                   = "de87dcxxxxfec671eexxxxxxxxx30accaxxxxa13" -> (known after apply)
+        # (3 unchanged attributes hidden)
+    }
+        Plan: 2 to add, 0 to change, 2 to destroy.
+
+It shows the following changes to output as below. You can ignore that message. Your infrastructure will not be affected.
+
+    Changes to Outputs:
+    ~ cis_dns_records     = {
+        ~ cis_imported_dns_records = [
+            ~ {
+                ~ file                 = "../../modules/dns/dns_records_2023-12-10T09:45:59Z.txt" -> (known after apply)
+                ~ id                   = "3:3:../../modules/dns/dns_records_2023-12-10T09:45:59Z.txt:e47cxxxxx7caf0xxxxx452395xxxxxfe:crn:v1:bluemix:public:internet-svcs:global:a/abxxxxx06xxxxa9cxxxxx44f5xxxxx0e:exxx8xx5-7xx1-4xx1-bxxe-2xxxxa2xxxxf::" -> (known after apply)
+                + records_added        = (known after apply)
+                + total_records_parsed = (known after apply)
+                },
+            ]
+    }
+
+### GLB submodule
+
+The [Global load balancer (GLB) submodule](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/glb/) provides Terraform resources to create and manage global load balancers in a CIS instance. It also allows to configure health checks, origin pools and proxy settings. For more information see [Global load balancer concepts](https://cloud.ibm.com/docs/cis?topic=cis-global-load-balancer-glb-concepts).
+
+When `glb_proxied` is set as `true`, then `ttl` is automatically set and cannot be updated.
+
+This means that when you run a `terraform plan` command after a successful `terraform apply`, the output shows that the `ttl` value requires an update, as shown in the following example. However, your infrastructure will not be affected and you can ignore that message.
+
+```
+# module.cis_glb[0].ibm_cis_global_load_balancer.cis_glb will be updated in-place
+    ~ resource "ibm_cis_global_load_balancer" "cis_glb" {
+        id               = "xxx:crn:v1:bluemix:public:internet-svcs:global:a/xxxe:xxx509::"
+        name             = "glb.***.com"
+        ~ ttl              = 0 -> 60
+```
+
+### WAF submodule
+
+The [Web Application Firewall (WAF) submodule](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/waf/) provides the Terraform resources to turn WAF on and off. CIS includes the default rule sets for WAF: the [OWASP rule set](https://cloud.ibm.com/docs/cis?topic=cis-waf-settings#owasp-rule-set-for-waf) and the [CIS rule set](https://cloud.ibm.com/docs/cis?topic=cis-waf-settings#cis-ruleset-for-waf). You can either enable or disable the WAF with these default rule sets.
+
+##  Activating DDOS protection
```

Comment:
Because this is at the same level as the submodules, I wonder if it deserves to go before that H2 section. My thinking is that DDos protection is more important to know than the submodules. 

But I'll leave that to you.

Reply:
I have moved the DDoS section up since it is more important than the submodule.